### PR TITLE
Implement a Select form field.

### PR DESCRIPTION
**Pull request by dhruvio**
Screen Shot 2022-02-11 at 12 25 00 AM

### CODE REVIEW

File: src/Elemental/Form/Validate.elm [hunk 1]
```diff
@@ -34,6 +35,11 @@ validate (Validator getErrors) input =
 -- CONSTRUCTING VALIDATORS
 
 
+new : (input -> List error) -> Validator error input
```
Comment (dwayne) [inline src/Elemental/Form/Validate.elm:38]: @dhruvio What's this for?

File: src/Elemental/View/Form/Field/Select.elm [hunk 2]
```diff
@@ -0,0 +1,246 @@
+module Elemental.View.Form.Field.Select exposing
```
Comment (dwayne) [inline src/Elemental/View/Form/Field/Select.elm:1]: We should add an examples directory to this project to be able to demo the various views. This way we can see and test them outside of any given project. What do you think?

**Review by dwayne — COMMENTED** [review]
Overall, it looks good. Thanks for the help.

File: src/Elemental/Form/Validate.elm (hunk 1 above)
Comment (dhruvio) [reply src/Elemental/Form/Validate.elm:38]: Spoke on the phone — commenting here for posterity. Adding an opaque value constructor for use with non-string input values.

**Pull request merged**